packages/world/src/modules/registration/tables/SystemRegistry.sol

```diff
@@ -8,6 +8,7 @@ import { SchemaType } from "@latticexyz/schema-type/src/solidity/SchemaType.sol"
 
 // Import store internals
 import { IStore } from "@latticexyz/store/src/IStore.sol";
+import { IStoreDynamicPartial } from "@latticexyz/store/src/IStoreDynamicPartial.sol";
```

Comment:
Is it an issue if we import the dynamics library in table libraries that don't need it?

Reply:
afaik no, solidity is good with this sort of tree shaking (specifically I was testing adding unused methods to libraries, and they didn't get included in bytecode; I assume imports are even easier to shake)